functions/addCustomUserData/source.js

```diff
@@ -0,0 +1,57 @@
+/**
+ * Should only be run from a trigger (args format matches trigger args)
+ */
+
+ /**
+  * For testsing:
+let tUser = { "data": { "email": "kylewilson52@gmail.com", "first_name": "Kyle", "last_name": "Wilson", "name": "Kyle Wilson" }, "id": "5e76c9c1b2f08c1062907727" };
+let input = {};
+input.user = tUser;
+exports(input);
```

Comment:
Is there a reason why you are not just passing in tUser to the function? That would be easier for the frontend to do rather than assign tUser to another object.

Reply:
yeah i know it's weird, but when the funciton is invoked from a trigger that's the format of the args.

as is, the front end shouldn't need to call it directly.